src/loki/search.cc

```diff
@@ -420,9 +446,17 @@ struct bin_handler_t {
                                  : candidate.sq_distance,
                              sq_tolerance, sq_max_distance);
       auto reach = get_reach(candidate.edge_id, candidate.edge);
-      PathLocation::PathEdge path_edge{candidate.edge_id, length_ratio, candidate.point,
-                                       distance,          side,         reach.outbound,
-                                       reach.inbound,     angle};
+      PathLocation::PathEdge path_edge{
```

Comment:
note that this does not work for node-snapped locations: we'd have to go through the bins again to find the bounding circle.